pkg/master/master.go

```diff
@@ -302,7 +303,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 	}
 
 	if c.ExtraConfig.EnableUISupport {
```

Comment:
Is EnableUISupport currently exposed in a flag?

Reply:
no, and access to `/ui` is not allowed as part of standard policy, either

Reply:
Didn't see it anywhere.